aioviberbot/api/api.py

```diff
@@ -30,9 +30,9 @@ def name(self):
     def avatar(self):
         return self._bot_configuration.avatar
 
-    async def set_webhook(self, url, webhook_events=None, is_inline=False):
+    async def set_webhook(self, url, webhook_events=None, send_name=True, send_photo=True, is_inline=False):
```

Comment:
Hi, thank you for PR!

Let's change order of function params here: `send_name` and `send_photo` should be after exisiting `is_inline` param to keep backward compatibility (if someone uses non-keywords args for this method)

Reply:
Got it, I'll remove it in next releases